src/chains/ton/deposit.ts

```
@@ -0,0 +1,47 @@
+import { toNano, TonClient } from "@ton/ton";
+import { Gateway } from "@zetachain/protocol-contracts-ton/dist/wrappers";
+import { z } from "zod";
+
+import { tonOptions } from "../../../types/ton.types";
+import { tonDepositParamsSchema } from "../../schemas/ton";
+import {
+  getGatewayAddress,
+  getWalletAndKeyPair,
+} from "../../../utils/ton.command.helpers";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)